introduction/architecture.md

```diff
@@ -111,6 +110,64 @@ while maintaining support for multiple indexes.
 For more on the motivation and design of TimescaleDB's adaptive space/time
 chunking, please see our [technical blog post][chunking].
 
+## TimescaleDB Clustering [](timescaledb-clustering) (PRIVATE BETA)
+
+TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives described above.
+In the multi-node topology, all nodes are TimescaleDB instances, but may play different roles
+depending on their responsibilities within a distributed database.
+
+The two roles that nodes can take on are:
+1) Access node
+2) Data node
+
+A front-end access node accepts all requests from the client, distributes the requests and queries
+appropriately to back-end data nodes, and aggregates the results received from the data nodes.
+Access nodes store cluster-wide information about the different data nodes as well as how
+chunks are distributed across those data nodes. Access nodes can also store regular, non-distributed
+hypertables, as well as regular PostgreSQL tables.
+
+Back-end data nodes store data associated with distributed hypertables. Data nodes are not
+aware of cluster-wide distribution policies, and otherwise look just as if they were single TimescaleDB
+instances. However, certain actions are blocked on data nodes to protect from accidentally executing operations on data nodes that might endanger the data consistency of distributed hypertables.
```

Comment:
split into two lines in markdown, please :)

Reply:
Done - should look better now!